microsetta_interface/templates/survey.jinja2

```diff
@@ -118,130 +190,219 @@
                     for (let entry of field.triggered_by) {
                         visible |= (model[entry['q_id']] === entry['response']);
                     }
+                    // If a triggered question is switched from visible to hidden, we're going to remove the answer from the model
+                    if(visible == false) {
+                        delete model[field.inputName];
+                    }
                     return visible;
                 }
             }
         }
     </script>
 {% endblock %}
 {% block breadcrumb %}
-    <li class="breadcrumb-item active" aria-current="page">{{ _('Participant Survey') }}</li>
+    <li class="breadcrumb-item"><a href="/accounts/{{account_id}}">{{ _('Dashboard') }}</a></li>
+    <li class="breadcrumb-item"><a href="/accounts/{{account_id}}/sources/{{source_id}}">{{ _('My Profile') }}</a></li>
+    <li class="breadcrumb-item active" aria-current="page">{{ survey_schema['groups'][0]['legend'] }}</li>
 {% endblock %}
 {% block content %}
-    <div id="form-progress">{{ _('Progress') }}</div>
-    <div class="panel panel-default">
-        <div class="panel-body">
-            <form id="survey_form" name="survey_form">
-                <vue-form-generator :schema="schema" :model="model" :options="formOptions" @validated="onValidated"></vue-form-generator>
-            </form>
+    <div class="container default-container">
+        <div class="row align-items-center mb-4">
+            <div class="col-xs-12 col-md-2 survey-header-icon">
+                <img src="/static/img/{{ survey_icon}}" />
+            </div>
+            <div class="col-xs-12 col-md-10">
+                <h1 class="h1-survey">{{ survey_schema['groups'][0]['legend'] }}</h1>
+                <span class="survey-subheader">{{ survey_est_minutes }} {{ _('minutes') }} - {{ survey_question_count }} {{ _('questions') }}</span>
+            </div>
+        </div>
+    </div>
+    <div class="container default-container">
+        <form id="survey_form" name="survey_form">
+            <vue-form-generator :schema="schema" :model="model" :options="formOptions" @validated="onValidated"></vue-form-generator>
+        </form>
+    </div>
+    <div class="container default-container">
+        <div class="row">
+            <div class="col-6">
+                {% if previous_survey != None %}
```

Comment:
Just saw this, here and line 234 I believe should use `is` and test against `none`. For context, the jinja2 docs advise using the lowercase variant (https://jinja.palletsprojects.com/en/3.0.x/templates/#expressions)